### DISCUSSION TITLE
How to custom auth provider change icon?

### DISCUSSION THREAD

**Discussion by Kir93** [strapi/strapi#7362] [Legacy Topics]
I am using strapi well.
I created a new auth provider, how can I change the icon?

**Comment by derrickmehaffy** [comment]
@soupette or @HichamELBSI any suggestions here?

**Comment by soupette** [comment]
@kir931028 what kind of provider's icon are your referring to?
If it's for a auth provider the icons are set [here](https://github.com/strapi/strapi/blob/9f4839028cb8cf939312383e4578c54f2df997fe/packages/strapi-plugin-users-permissions/config/functions/bootstrap.js#L35-L42)

**Comment by Kir93** [comment]
@soupette I looked at your documentation and created a new Provider.
So I want to use a new icon for that Provider in the strapi admin panel.
```
myProvider: {
      enabled: false,
      icon: "custom icon", // here
      key: "",
      secret: "",
      callback: `${strapi.config.server.url}/auth/myProvider/callback`,
    },
```

**Comment by lauriejim** [comment]
Hello, the file mentioned by Soupette is used for the initialization of the grant data into the core store.
Then everything is managed into the database.
You will have to check in your database, the `core_store` table and search for the line with `plugin_users-permissions_grant` as key.
The value will be an object as mentioned by Soupette. Update the value to match your need.

**Comment by Kir93** [comment]
@lauriejim Thank you for the reply.
But I don't understand what you are saying.
Are you saying that if you want to change the icon, you have to edit other parts?
Any examples would be appreciated.
Thank you.

**Reply by lauriejim** [reply]
The file you updated is used during the first start of your application when the database info doesn't exist.
Here you are trying to do your action AFTER having started your app.
So you will have to update the database info I mentioned instead of the file.

**Reply by Kir93** [reply]
@lauriejim Thank you for your reply!
Thank you so much for your kind answer.
Have a nice day :)

**Reply by lauriejim** [reply]
Thanks
